dimod/binary_quadratic_model.py

```diff
@@ -891,6 +891,40 @@ def scale(self, scalar):
         except AttributeError:
             pass
 
+    def normalize(self, a=1):
```

Comment:
More descriptive name for `a`? Also, would asymmetric range make sense? `[a, b]`?